js/reducers/passageReducer.js

```diff
@@ -12,39 +12,56 @@
  * To add a new reducer, add a file like this to the reducers folder, and
  * add it in the rootReducer.js.
  */
+import assignToEmpty from '../utils/assign';
 
-import { NEXT_VERSE, PREVIOUS_VERSE, ENABLE_RECALL, DISABLE_RECALL } from '../constants/AppConstants';
+import { NEXT_VERSE, PREVIOUS_VERSE, ENABLE_RECALL, DISABLE_RECALL, CHANGE_MODE } from '../constants/AppConstants';
 import * as passage from '../passage'
-import assignToEmpty from '../utils/assign';
 
 const verses = passage.verses();
-function verseByIndex(index) {
-  const verse = verses[index];
-  return {
-    verseMetadata: verse.book + ' ' + verse.chapter + ':' + verse.verse,
-    verseText: verse.text,
-    verseIndex: index
-  }
-}
-const initialState = assignToEmpty(verseByIndex(0), {
-  verseCount: verses.length
+
+const initialState = assignToEmpty({
+  activeVerse: 0,
+  totalVerses: verses.length,
+  verses     : verses,
+  mode       : 'multi'
```

Comment:
Yup! It should. Missed one when I was upgrading everything else to the constant.